tests/test_common_gpu.py

```diff
@@ -1572,6 +1573,28 @@ def test_vera_add_new_adapter_does_not_change_device(self, mlp):
         assert model.lin0.vera_A.other.device.type == "cuda"
         assert model.lin0.vera_lambda_d.other.device.type == "cuda"
 
+    def test_vblora_add_new_adapter_does_not_change_device(self, mlp):
+        # same as first test, but using VBLoRA
+        config = VBLoRAConfig(target_modules=["lin0"], vector_length=2)
+        model = get_peft_model(mlp, config)
+        model = model.cuda()
+        model.lin0.vblora_logits_A.cpu()
```

Comment:
Let's also check `vblora_logits_B`.

Reply:
Sure. Added some lines to check `vblora_logits_B`.